apps/v3-subgraph/schema.graphql

```diff
@@ -103,10 +107,26 @@ type Proposal @entity {
   sponsored: Boolean!
   "address that sponsored the proposal"
   sponsor: Bytes!
+  "transaction hash of the proposal sponsor"
+  proposalSponsorTxHash: Bytes!
```

Comment:
This should probably be nullable as we won't have data for that field when saving the proposal upon submission.